### ISSUE TITLE
Bone Marrow Cells Classification using Deep Learning

### ISSUE THREAD

**Issue by adi271001** [abhisheks008/DL-Simplified#376]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Bone Marrow Cells Classification using EffecientNetb5 </br>
:red_circle: **Aim** :  To classify Bone Marrow Cells using EfficientnetB5 </br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/andrewmvd/bone-marrow-cell-classification</br>
:red_circle: **Approach** : Model uses EfficientNetB5 as base model and adds 4 extra layers batch normalization layer , dense layer , dropout layer , dense_1 layer.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name :  Aditya D
- GitHub Profile Link : https://www.github.com/adi271001
- Email ID : ibetanibex@gmail.com
- Participant ID (if applicable):
- Approach for this Project :
1. Import Libraries
2. Perform Preprocessing of Data
3. Creating all the required functions
4. Split Data into test and train
5. Build the EfficientNetB5 model
6. Train the model
7. Predict on test data
8. Recording the accuracy and plots to check over/underfitting
9. Saving the Model and predicition results in csv

- What is your participant role? (Mention the Open Source program) OSEN Participant

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Hi @adi271001 thanks for showing interest in Deep Learning Simplified. As per the code of conduct, each contributor can work on one issue at a time, on which issue you want to work first?

**Comment by adi271001** [comment]
@abhisheks008 i want to work on both i'll work on the lung and colon one first after i finsih that i'll work on this

**Comment by adi271001** [comment]
@abhisheks008  can you assign this to me

**Comment by abhisheks008** [comment]
Sure. Issue assigned to you @adi271001

**Comment by abhisheks008** [comment]
This issue will not be considered under CollabCode Open Source event organized by OSEN.

**Comment by Dhruv127** [comment]
Can you kindly assign me the issue?

**Comment by CoderOMaster** [comment]
Hi @abhisheks008 can I contribute in this?
Thank you

**Comment by abhisheks008** [comment]
Hi @CoderOMaster are you part of SWOC? If yes, you can comment out here once the SWOC event officially starts.

**Comment by CoderOMaster** [comment]
@abhisheks008 I couldn't register on for swoc,but still possible to contribute?

**Comment by abhisheks008** [comment]
Right now, this project repository is going to be part of Social Winter of Code. If you want to contribute you can join it as a Participant.

**Comment by CoderOMaster** [comment]
Registration just closed a few days ago,how can I join as participant?
Thank you

**Comment by abhisheks008** [comment]
You can connect with the organizing team of SWOC by writing an email on socialwinterofcodee@gmail.com

**Comment by the-silent-geek** [comment]
Full name : D. Uday Kiran
GitHub Profile Link : https://github.com/the-silent-geek
Email ID : udaykir3911@gmail.com
Approach for this Project : To implement EfficientnetB5
What is your participant role? social winter of code 23

**Comment by abhisheks008** [comment]
Also implement MobileNet, ResNet and VGG.
@the-silent-geek 

Issue assigned to you.
